contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to {{Project-Name}}
+
+We welcome everyone to contribute what they can. Whether you are brand new, just want to contribute a little bit, or want to contribute a lot there is probably something you can help out with.
```

Comment:
We should note that we'll provide labels on issues for `good_first_issue` items (https://build.prestashop-project.org/news/a-definition-of-the-good-first-issue-label), making it easy to get started.

Reply:
We can maybe even embed the label. I'll make sure we have it at the org level.